config/environments/production.rb

```diff
@@ -72,14 +72,8 @@
   # the I18n.default_locale when a translation cannot be found).
   config.i18n.fallbacks = true
 
-  # Send deprecation notices to registered listeners.
-  config.active_support.deprecation = :notify
-
-  # Log disallowed deprecations.
-  config.active_support.disallowed_deprecation = :log
-
-  # Tell Active Support which deprecation messages to disallow.
-  config.active_support.disallowed_deprecation_warnings = []
+  # Don't log any deprecations.
+  config.active_support.report_deprecations = false
```

Comment:
why are we removing the deprecation warnings?

Reply:
This is to leave this file with the config that Rails 7 would generate from scratch. Check https://github.com/rails/rails/blob/main/railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt#L88

Reply:
well, that makes sense when you start from scratch but when you are migrating from an older codebase I think it could hide some deprecation issues